File: android/src/main/java/name/zeno/android/tint/TintableDrawableView.java
package name.zeno.android.tint;

import android.content.res.ColorStateList;
import android.graphics.PorterDuff;
import android.support.annotation.Nullable;

/**
 * Create Date: 16/6/23
 *
 * @author 陈治谋 (513500085@qq.com)
 */
public interface TintableDrawableView
{
  @Nullable ColorStateList getSupportDrawableTintList();

  void setSupportDrawableTintList(@Nullable ColorStateList tint);

  void setSupportDrawableTintMode(@Nullable PorterDuff.Mode tintMode);

  @Nullable PorterDuff.Mode getSupportDrawableTintMode();
}

File: zako/src/main/java/cn/izeno/android/tint/TintableDrawableView.kt
package cn.izeno.android.tint

import android.content.res.ColorStateList
import android.graphics.PorterDuff

/**
 * Create Date: 16/6/23
 *
 * @author 陈治谋 (513500085@qq.com)
 */
interface TintableDrawableView {
  var supportDrawableTintList: ColorStateList?

  var supportDrawableTintMode: PorterDuff.Mode?
}
